Scrabble/Game Objects/Board.cs

```diff
@@ -1,23 +1,857 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿//-----------------------------------------------------------------------
+// <copyright file="Board.cs" company="Scrabble Project Developers">
+//     Copyright (c) Scrabble Project Developers. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
 
 namespace Scrabble.Game_Objects
-{  
-    // Board class handles the list of tiles, and plays made to the board.
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
 
-    class Board
+    /// <summary>
+    /// The board is a 15x15 grid of <see cref="GameBoardSquare"/>s to which plays can be added.
+    /// </summary>
+    public class Board
     {
-        //Contains the list of board tiles.
-        private List<GameBoardSquare> SquaresList;
-        public Board() { }
         /// <summary>
-        /// Applies a Play to the tiles in SquaresList.
-        /// Play parameter might need to be traced to its location once created.
+        /// A list of <see cref="GameBoardSquare"/>s.
         /// </summary>
-        /// <param name="play">Receives a play and applys that to tiles.</param>
-        public void AddPlayToBoard(Play play) { }
+        private List<GameBoardSquare> squaresList;
+
+        /// <summary>
+        /// A 15x15 grid made up of the squares in squaresList.
+        /// </summary>
+        private GameBoardSquareGrid boardGrid;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Board" /> class.
+        /// </summary>
+        public Board()
+        {
+            this.squaresList = new List<GameBoardSquare>();
+
+            for (int y = 0; y < 15; ++y)
+            {
+                for (int x = 0; x < 15; ++x)
+                {
+                    this.squaresList.Add(new GameBoardSquare(x, y, this.GetWordMultiplier(x, y), this.GetLetterMultiplier(x, y)));
+                }
+            }
+
+            this.boardGrid = new GameBoardSquareGrid(this.squaresList);
+        }
+
+        /// <summary>
+        /// Tells whether a <see cref="Play"/> can legally be placed onto the board.
+        /// </summary>
+        /// <param name="play">The <see cref="Play"/> being checked.</param>
+        /// <returns>True if play can be added to the board. Returns false otherwise.</returns>
+        public bool PlayIsValid(Play play)
+        {
+            if (play.GetParallelListLength() == 0)
```

Comment:
Parallel list length should never be zero it would throw an EmptyParrallelList() error.